cli/running/running_test_helpers.go

```diff
@@ -6,5 +6,5 @@ import "time"
 func waitProcessStart() {
 	// We need to wait for the new process (tarantool instance) to set handlers.
 	// It is necessary to update for more correct synchronization.
-	time.Sleep(100 * time.Millisecond)
+	time.Sleep(1000 * time.Millisecond)
```

Comment:
As I can see, we know the console socket path. Can we wait for the socket to appear instead of increasing timeout?

Reply:
As far as I remember, socket appearance!=instance start. It is created before instance is fully started. So, I think we could check for socket + some timeout / stay as is.